src/test/scala/simulacrum/typeclass.scala

```diff
@@ -222,11 +222,14 @@ class TypeClassTest extends WordSpec with Matchers {
       }
 
       "lifted type argument in method bodies are supported" in {
-        @typeclass trait Monoid[A] { def append(x: A, y: A): A; def id: A }
-        @typeclass trait Foldable[F[_]] {
-          def foldLeft[A, B](fa: F[A])(b: B)(f: (B, A) => B): B
-          def concatenate[A: Monoid](fa: F[A]): A = foldLeft(fa)(Monoid[A].id)((acc, a) => Monoid[A].append(acc, a))
+        object typeClasses {
```

Comment:
Defining these traits in an object (instead of directly inside of the method) is necessary for 2.10 because of something related to [SI-6231](https://issues.scala-lang.org/browse/SI-6231) that I only half understand at best.